src/applications/vaos/new-appointment/components/PreferredDatePage.unit.spec.jsx

```diff
@@ -112,16 +112,9 @@ describe('VAOS Page: PreferredDatePage', () => {
   });
 
   it('should submit with valid data', async () => {
-    const maxMonth = moment()
-      .add(395, 'days')
-      .format('M');
-    const maxDay = moment()
-      .add(395, 'days')
-      .format('DD')
-      .replace(/\b0/g, '');
-    const maxYear = moment()
-      .add(395, 'days')
-      .format('YYYY');
+    const maxMonth = format(addDays(new Date(), 395), 'M');
+    const maxDay = format(addDays(new Date(), 395), 'dd').replace(/\b0/g, '');
```

Comment:
So just use the format string 'd' instead?

Reply:
done.